test_problems/rankine_democxx/rankine.cpp

```diff
@@ -82,7 +77,7 @@ int main()
     try {
         return openRankine(0, 0);
     } catch (CanteraError& err) {
-        std::cout << err.what() << std::endl;
+        printf("%s", err.what());
```

Comment:
Why is this preferable to using `std::cout`?

Reply:
In this change and changing `size_t` to `int`, I was making the test code to be the same as the sample. I'm not sure why the code is nearly duplicated but I didn't want to modify SConscruct to copy the files around, so I made them as close as possible 🤷‍♂